h2/config.py

```diff
@@ -70,6 +86,11 @@ class H2Configuration(object):
         be skipped, and allow the object to receive headers that may be illegal
         according to RFC 7540. Defaults to ``True``.
     :type validate_inbound_headers: ``bool``
+
+    :param logger: A logger that conforms to the requirements for this module,
+        those being no I/O and no context switches, which is needed in order
+        to run in asynchronous operation.
+    :type logger: ``logging.Logger``
```

Comment:
We should add a `versionadded` parameter to the docstring, i.e.:

```
    :param logger: A logger that conforms to the requirements for this module,
        those being no I/O and no context switches, which is needed in order
        to run in asynchronous operation.

        .. versionadded:: 2.6.0

    :type logger: ``logging.Logger``
```

You can see an example of what that looks like in the rendered documentation here: https://python-hyper.org/h2/en/stable/api.html#h2.connection.H2Connection